src/Data/Global.lua

```diff
@@ -284,3 +284,5 @@ GlobalCache = {
 	useFullDPS = false,
 	numActiveSkillInFullDPS = 0,
 }
+
+POESESSID = ""
```

Comment:
This could be better suited to being saved in Settings.xml.  Obviously it can expire, but maybe we can prompt for an update then?

Reply:
If you do that, it will get saved to the Build.xml by default. You would be sharing people's POESESSID as part of their build with pastebin, etc.

Reply:
Not Build.xml, Settings.xml.  A user's settings doesn't get stored with a build.